evennia/commands/default/general.py

```diff
@@ -387,48 +387,72 @@ class CmdGet(COMMAND_DEFAULT_CLASS):
     Usage:
       get <obj>
 
-    Picks up an object from your location and puts it in
-    your inventory.
+    Picks up an object from your location and puts it in your inventory.
     """
 
     key = "get"
     aliases = "grab"
-    locks = "cmd:all();view:perm(Developer);read:perm(Developer)"
+    locks = "cmd:all()"
     arg_regex = r"\s|$"
 
+    def parse(self):
+        super().parse()
+        self.number = 0
+        if self.args:
+            # check for numbering
+            count, *args = self.args.split(maxsplit=1)
+            # we only use the first word as a count if it's a number and
+            # there is more text afterwards
+            if args and count.isdecimal():
+                self.number = int(count)
+                self.args = args[0]
```

Comment:
This same block is repeated over and over.
- Maybe commands with item targeting like this should all share the same parent class. 
- Another option would be to add this kind of functionality to the `object.search()`, but possibly that makes it harder to customize.